CMakeLists.txt

```diff
@@ -757,15 +757,19 @@ if(WIN32)
     install(
         FILES
         OpenAL32.dll
-        boost_date_time-vc90-mt-1_44.dll
-        boost_filesystem-vc90-mt-1_44.dll
-        boost_iostreams-vc90-mt-1_44.dll
-        boost_python-vc90-mt-1_44.dll
-        boost_regex-vc90-mt-1_44.dll
-        boost_serialization-vc90-mt-1_44.dll
-        boost_signals-vc90-mt-1_44.dll
-        boost_system-vc90-mt-1_44.dll
-        boost_thread-vc90-mt-1_44.dll
+        boost_chrono.dll
+        boost_date_time.dll
+        boost_filesystem.dll
```

Comment:
Is there some way to tell MSVC to link against these filenames instead of the 
`boost_filesystem-vc141-mt-x32-1_66.lib` style that it seems to expect by default? I'm not having any luck googling.

Reply:
I will take a look at this, but the term you want to search for is boost auto-linking.

Reply:
Should be fixed with the latest commit on this branch.